### ISSUE TITLE
Add support for French and German languages

### ISSUE THREAD

**Issue by ifabijanovic** [ifabijanovic/swtor-holonet#51]

**Comment by 1nurion** [comment]
"tab_forum" = "Forum";
"tab_dulfy" = "Dulfy";
"tab_settings" = "Einstellungen";

"alert_dulfy_title" = "Dulfy";
"alert_dulfy_dismiss" = "Verbergen";
"alert_dulfy_confirm" = "Zeigen";

"alert_network_error_title" = "Netzwerkfehler";
"alert_network_error_body" = "Beim Laden der Daten ist ein Fehler aufgetreten. Möchtest du es erneut versuchen?";
"alert_network_error_dismiss" = "Nein";
"alert_network_error_confirm" = "Ja";

"alert_maintenance_title" = "Wartungsarbeiten";
"alert_maintenance_body" = "SWTOR.com ist aufgrund von planmäßigen Wartungsarbeiten zur Zeit nicht verfügbar.";
"alert_maintenance_confirm" = "OK";

"theme_dark_name" = "Dunkel";
"theme_light_name" = "Hell";

"text_size_small_name" = "Klein";
"text_size_medium_name" = "Mittel";
"text_size_large_name" = "Groß";

"forum_language_english_name" = "Englisch";
"forum_language_french_name" = "Französisch";
"forum_language_german_name" = "Deutsch";

"forum_developer_tracker_thread_title" = "Entwicklernews";
"forum_root_title" = "Forum";
"forum_thread_title" = "Thread";
"forum_post_title" = "Post";
"forum_categories_section_title" = "Kategorien";
"forum_threads_section_title" = "Threads";
"forum_thread_replies_label" = "A: %d, H: %d";

"alert_notification_request_access_title" = "Mitteilungen";
"alert_notification_request_access_body" = "Hey, möchtest du Mitteilungen von HoloNet und Dulfy erhalten?";
"alert_notification_request_access_dismiss" = "Nein";
"alert_notification_request_access_confirm" = "Ja";

"settings_title" = "Einstellungen";
"settings_section_messages_title" = "NACHRICHTEN";
"settings_section_display_title" = "ANZEIGE";
"settings_section_feedback_title" = "FEEDBACK";
"settings_section_legal_title" = "IMPRESSUM";
"settings_notifications_label" = "Mitteilungen";
"settings_notifications_enabled_value" = "Aktiviert";
"settings_notifications_disabled_value" = "Deaktiviert";
"settings_forum_language_label" = "Forensprache";
"settings_theme_label" = "Theme";
"settings_text_size_label" = "Schriftgröße";
"settings_contact_label" = "Kontakt";
"settings_report_bug_label" = "Fehler melden";
"settings_disclaimer_label" = "Haftungsausschluss";
"settings_privacy_policy_label" = "Datenschutzerklärung";
"settings_license_label" = "Lizenz";

"alert_email_not_available_title" = "Fehler";
"alert_email_not_available_body" = "Es scheint als wäre keine Email auf diesem Gerät konfiguriert.";
"alert_email_not_available_confirm" = "OK";



#####     COMMENTS     #####
General note: If you have trouble with special characters like ä ü ö ß let me know and I will find a workaround to avoid those characters - Possibly affected lines are 10, 15, 23, 26, 38, 52, 56, 60


Line 10 alternative - shorter and no special characters, very generic and not as polite
	"Fehler beim Laden der Daten. Erneut versuchen?"


Line 15 alternative - shorter and no special characters
	"SWTOR.com ist aufgrund von Wartungsarbeiten zur Zeit nicht erreichbar."


Line 25 - 27 alternative suggestion for EN/FR/GER version of your App
	"forum_language_english_name" = "English";
	"forum_language_french_name" = "Français";
	"forum_language_german_name" = "Deutsch";

	Consider to use the code mentioned above instead for EN/FR/GER version of your App because if someone wants to look for the German forum he most likely will search for "Deutsch" and not "German". This example above would be localization while the provided strings for 25-27 are basically translations. The choice is yours.


Line 37, 38, 47 Terminology iOS vs. Android
	Please note that for the term "Notifications" iOS sticks in German to "Mitteilungen" while Android uses "Benachrichtigungen". Different words with same meaning (Synonyms). Don't know how strict the control is when you want to publish your App on either platform but I thought it was worth mentioning because of the different terminology/wording Apple/Google use.


Line 56 alternative - shorter
	Datenschutz


Line 57
	If it is more than 1 license please use "Lizenzen" instead


Line 60 alternative - shorter and no special characters, very generic and not as polite
	"Keine Email auf diesem Geraet konfiguriert."

**Comment by ifabijanovic** [comment]
This is excellent, thank you very much! I don't think the special characters will be a problem. 

I forgot one more text I would like localised if its not a problem - the App description shown in App Store:

```
Browse the official Star Wars: The Old Republic community forums in a easy to use, mobile friendly way. Connect to the popular Dulfy.net site and get notifications when new content is available.

Features:
- Browse through all available sub forums.
- Read forum posts using a clear UI.
- Browse the popular Dulfy.net fansite which contains guides and walkthroughs.
- Get notifications from Dulfy.net when new content is available.

Disclaimer: SWTOR HoloNet is an application developed by SW:TOR fans and is not affiliated with Disney, Lucasfilm, EA or Bioware.
```

**Comment by ifabijanovic** [comment]
German localisation implemented in #60

**Comment by 1nurion** [comment]
Durchsuche die offiziellen Star Wars: The Old Republic Community-Foren mobil auf einfache Art und Weise. Verbinde dich mit der beliebten Dulfy.net Webseite und erhalte umgehend Mitteilungen sobald neue Inhalte verfügbar sind.

Features:
- Durchsuche alle verfügbaren Unterforen.
- Lies die Forenbeiträge auf einer übersichtlichen Benutzeroberfläche. 
- Durchsuche die beliebte Dulfy.net Fanseite, die dich mit Guides und Walkthroughs versorgt.
- Erhalte Mitteilungen von Dulfy.net sobald neue Inhalte verfügbar sind.

Haftungsausschluss: SWTOR HoloNet ist eine Applikation entwickelt von SW:TOR Fans und weder Teil von, noch verbunden mit Disney, Lucasfilm, EA oder Bioware.

**Comment by Soontiir** [comment]
"tab_forum" = "Forum";
"tab_dulfy" = "Dulfy";
"tab_settings" = "Paramètres";

"alert_dulfy_title" = "Dulfy";
"alert_dulfy_dismiss" = "Cacher";
"alert_dulfy_confirm" = "Afficher";

"alert_network_error_title" = "Erreur réseau";
"alert_network_error_body" = "Une erreur s'est produite lors du chargement des données. Voulez-vous réessayer?";
"alert_network_error_dismiss" = "Non";
"alert_network_error_confirm" = "Oui";

"alert_maintenance_title" = "Maintenance";
"alert_maintenance_body" = "SWTOR.com est actuellement indisponible pendant que la maintenance programmée est effectuée.";
"alert_maintenance_confirm" = "OK";

"theme_dark_name" = "Sombre";
"theme_light_name" = "Clair";

"text_size_small_name" = "Petit";
"text_size_medium_name" = "Moyen";
"text_size_large_name" = "Grand";

"forum_language_english_name" = "Anglais";
"forum_language_french_name" = "Français";
"forum_language_german_name" = "Allemand";

"forum_developer_tracker_thread_title" = "Fil du développeur";
"forum_root_title" = "Forum";
"forum_thread_title" = "Discussion";
"forum_post_title" = "Message";
"forum_categories_section_title" = "Catégories";
"forum_threads_section_title" = "Discussions";
"forum_thread_replies_label" = "R: %d, A: %d";

"alert_notification_request_access_title" = "Notifications";
"alert_notification_request_access_body" = "Hey, voulez-vous recevoir des notifications de l'HoloNet et de Dulfy?";
"alert_notification_request_access_dismiss" = "Non";
"alert_notification_request_access_confirm" = "Oui";

"settings_title" = "Paramètres";
"settings_section_messages_title" = "MESSAGES";
"settings_section_display_title" = "AFFICHER";
"settings_section_feedback_title" = "FEEDBACK";
"settings_section_legal_title" = "MENTIONS LEGALES";
"settings_notifications_label" = "Notifications";
"settings_notifications_enabled_value" = "Autoriser";
"settings_notifications_disabled_value" = "Désactiver";
"settings_forum_language_label" = "Langue du Forum";
"settings_theme_label" = "Thème";
"settings_text_size_label" = "Taille du texte";
"settings_contact_label" = "Contacter";
"settings_report_bug_label" = "Rapport de bug";
"settings_disclaimer_label" = "Avertissement";
"settings_privacy_policy_label" = "Politique de confidentialité";
"settings_license_label" = "Licence";

"alert_email_not_available_title" = "Erreur";
"alert_email_not_available_body" = "Il semble que les emails ne sont pas configurés sur cet appareil.";
"alert_email_not_available_confirm" = "OK";

_______________________________________________________________________________________________________

So in lines 10 and 38, I used "Voulez-vous", which is a more "formal/professionnal/polite" way to talk than "Veux-tu". If you want you can switch those lines with those :

"alert_network_error_body" = "Une erreur s'est produite lors du chargement des données. Veux-tu réessayer?";

"alert_notification_request_access_body" = "Hey, veux-tu recevoir des notifications de l'HoloNet et de Dulfy?";

In line 35 (V for views/number of views), it would be better to use V for "vues/nombre de vues", but in the french forum it says "Affichages", so I used "A"

_______________________________________________________________________________________________________

Parcourez facilement sur mobile les forums officiels Star Wars: The Old Republic. Connectez-vous au célèbre site Dulfy.net et recevez des notifications lorsque de nouveaux contenus sont disponibles.

Caractéristiques: 
- Parcourez tous les sous-forums disponibles. 
- Lire les publications du forum en utilisant une interface claire. 
- Parcourez le célèbre fansite Dulfy.net qui contient des guides et des astuces. 
- Recevez des notifications de Dulfy.net lorsque de nouveaux contenus sont disponibles.

Avertissement: SWTOR HoloNet est une application développée par les fans de SW: TOR et n'est pas affiliée à Disney, Lucasfilm, EA ou Bioware.

_______________________________________________________________________________________________________

**Comment by ifabijanovic** [comment]
French localisation implemented in #75
